applications/CompressiblePotentialFlowApplication/custom_conditions/potential_wall_condition.cpp

```diff
@@ -254,11 +254,11 @@ void PotentialWallCondition<TDim, TNumNodes>::load(Serializer& rSerializer)
 // private operators
 
 template <unsigned int TDim, unsigned int TNumNodes>
-inline Element::Pointer PotentialWallCondition<TDim, TNumNodes>::pGetElement() const
+inline GlobalPointer<Element> PotentialWallCondition<TDim, TNumNodes>::pGetElement()
 {
-    KRATOS_ERROR_IF(!mpElement.lock())
+    KRATOS_ERROR_IF(mpElement.get() == nullptr)
```

Comment:
If the pGetElement method is defined as constant, this throws an error in the get() call:

```
error: no matching function for call to ‘Kratos::GlobalPointer<Kratos::Element>::get() const
     KRATOS_ERROR_IF(mpElement.get() == nullptr)
```